apps/internal/base/base.go

```diff
@@ -89,6 +89,7 @@ type AuthResult struct {
 	ExpiresOn      time.Time
 	GrantedScopes  []string
 	DeclinedScopes []string
+	IsFromCache    bool
```

Comment:
Other MSALs added a new field `token_source` which value can be `identity_provider`, `cache` and `broker`.

Reply:
Thank you for the comment, I looked into msal .Net and msal java and updated the pullrequest with this
As, we don't have support for broker atm, I did not add it. 
When in Future we add that we can update the `TokenSource`